BlackNectar/BlackNectar/AppDelegate.swift

```diff
@@ -27,6 +42,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func applicationDidEnterBackground(_ application: UIApplication) {
         // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
         // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
+        
+        AromaClient.sendLowPriorityMessage(withTitle: "App Entered Background")
+        ImageCache.default.clearMemoryCache()
```

Comment:
It is good to clear the cache when the user leaves the app to reduce the memory load.